pyproject.toml

```diff
@@ -83,19 +84,21 @@ homepage = "https://mne.tools/mne-bids-pipeline"
 repository = "https://github.com/mne-tools/mne-bids-pipeline"
 changelog = "http://mne.tools/mne-bids-pipeline/changes.html"
 
-[build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2", "wheel"]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools_scm]
-tag_regex = "^(?P<prefix>v)?(?P<version>[0-9.]+)(?P<suffix>.*)?$"
-version_scheme = "release-branch-semver"
-
-[tool.setuptools.packages.find]
-exclude = ["false"]  # on CircleCI this folder appears during pip install -ve. for an unknown reason
+[tool.hatch.version]
+source = "vcs"
+raw-options = { version_scheme = "release-branch-semver" }
 
-[tool.setuptools.package-data]
-"mne_bids_pipeline.steps.freesurfer.contrib" = ["version"]
+[tool.hatch.build]
+exclude = [
+    "/.*",
+    "/codecov.yml",
+    "**/tests",
+    "/docs",
```

Comment:
@ofek I'm explicitly excluding `/docs` here, but `docs/source/examples/gen_examples.py` is still being packaged when I build the sdist. Any idea why that's happening?

Reply:
Ah! There's an entry in `.gitignore`:

```
!docs/source/examples/gen_examples.py
```

Wonder why we have that one … but this explains it.